translations/mgcp.js

```diff
@@ -76,7 +76,7 @@ mgcp = {
 
   // validateAttrs: Clean up the supplied attr list by dropping anything that should not be part of the
   //        feature
-  validateAttrs: function(geometryType,attrs) {
+  validateAttrs: function(geometryType,attrs,notUsed,transMap) {
```

Comment:
Tag preservation.
notUsed == list of tags that haven't been used during the translation
transMap == the translation (tag & value) for each if the attributes.

If validation drops an attribute due to it not being in the spec for that feature, the tag and value for that attribute get pushed onto the notUsed list to preserve them.